### PR TITLE
Adds ResumeBot

### PR DESCRIPTION
**Pull request by Lontronix**
Resumes can now send a slack message once per day with a list of users that uploaded users in the past 24 hours. If no users uploaded a resume on a given day, no messages will be sent

CleanShot 2023-02-11 at 21 35 34@2x

### CODE REVIEW

**Comment by Lontronix** [comment]
oh and I also did some testing in my own s3 bucket and dev database so I'm pretty confident this shouldn't break shit

File: routes/upload.js [hunk 1]
```diff
@@ -42,13 +43,14 @@ router.post('/',
       }
       // if not found, upload it
       var filename = req.body.title || req.file.originalname;
-      var author = req.user._json.preferred_username;
+      var authorUsername = req.user._json.preferred_username;
+      var authorFullname = req.user._json.name;
```
Comment (jabbate19) [inline routes/upload.js:47]: Unused Variable?

File: db/repos/resumes.js [hunk 2]
```diff
@@ -52,6 +52,14 @@ class ResumesRepository {
...
+    /**
+    Fetches a list of users that posted a resume in the last 24 hours
+    */
+    fetchRecentUploders() {
+        return this.db.any(sql.dailyDigest);
+    }
+
 }
 
 module.exports = ResumesRepository;
```
Comment (jabbate19) [inline db/repos/resumes.js:65]: Isn't your fault, but could you add a newline at this file?

**Review by jabbate19 — CHANGES_REQUESTED** [review]
I'm also just curious, how does this start the job? Is it started when you require the module?

File: routes/upload.js (hunk 1 above)
Comment (Lontronix) [reply routes/upload.js:47]: yes, good catch

**Review by jabbate19 — APPROVED** [review]
:shipit:

**Pull request merged**